pages/_app.js

```diff
@@ -33,6 +33,7 @@ function MyApp({ Component, pageProps }) {
           content={process.env.NEXT_PUBLIC_BUILD_DATE}
         />
         <meta name="og:buildid" content={process.env.NEXT_PUBLIC_TC_BUILD} />
+        <script src="//assets.adobedtm.com/be5dfd287373/0127575cd23a/launch-913b1beddf7a-staging.min.js"></script>
```

Comment:
We probably want to put the link in an env and then check to see if it exists before adding it to the page.
It should look something like:
```
...
{
  if (process.env.ADOBE_ANALYTICS_URL){
    <script src=${process.env.ADOBE_ANALYTICS_URL}></script>
  }
}
...
```

We don't want to collect analytics while developing, so locally we won't have that env, but it would exist in our Azure WebApp. This also means we don't have to make any code changes when going to prod.

Reply:
This is a great point. @EphraimAssi I'm wondering if there is any mention of recommended practices in respect to this within the install documentation?

Reply:
@will0684 They did mention whitelisting some domains that may block the scripts but nothing specific on the implementation.